src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp

```diff
@@ -0,0 +1,85 @@
+<%@ page pageEncoding="UTF-8" contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
```

Comment:
We always put attributes in the model here. Please have a look into other jsp files.